app/views/play/level/tome/SpellView.coffee

```diff
@@ -694,6 +711,30 @@ module.exports = class SpellView extends CocoView
     return if @destroyed
     Backbone.Mediator.publish 'tome:hide-problem-alert', {}
 
+  putSpade: () ->
+    # This happens when 1) The user has taken longer than 10 minutes, 2) the Contact button is pressed.
+    # Let's check to make sure SPADE exists.
+    if @spade
```

Comment:
Just do `return unless @spade` so that we don't have an extra level of indentation and so that the "happy path" ("golden path") of the default case (we do have `@spade`) is the "normal", un-indented one, and exceptions are the indentations.

... but also I think we will now always have `@spade`, right? So we don't need this check either.